### DISCUSSION TITLE
Support loading only the audio waveform without loading the audio.

### DISCUSSION THREAD

**Discussion by lmy1991** [katspaugh/wavesurfer.js#3160] [Q&A]
If a website needs to display multiple audio lists, the current practice requires downloading multiple audio files simultaneously. However, this is not necessary. I propose to load only the audio waveform and defer loading the audio files until the user actually needs to play them.
 I hope the official team can consider adopting this approach. Thank you.

**Comment by katspaugh** [comment]
Pass an empty string to prevent loading any audio.
`wavesurfer.load(“”, peaks, duration)`
